BookingsApi/BookingsApi.DAL/Queries/GetBookingsByCaseTypesQuery.cs

```diff
@@ -164,6 +179,23 @@ private IQueryable<VideoHearing> GetHearingsWithoutJudge(IQueryable<VideoHearing
 
             return videoHearings.AsQueryable();
         }
+        
+        private IQueryable<VideoHearing> GetHearingsNotAllocated(IQueryable<VideoHearing> hearings)
+        {
+            var videoHearings = new List<VideoHearing>();
```

Comment:
Could we add a test for a non-empty lis of hearings?

Reply:
test added

Reply:
done